docs/ServerSide.md

```diff
@@ -50,7 +50,7 @@ The `.code-workspace` file is a JSON file which you can edit directly, as descri
    - The value following `/` specifies the name of the server.
    - The value following `:` specifies the namespace (lowercase).
 
-The string `isfs` which appears in the **uri** for folders configured for server-side editing is an abbreviation created by InterSystems which stands for **InterSystems File Service**. It implements the VS Code [FileSystemProvider API](https://code.visualstudio.com/api/references/vscode-api#FileSystemProvider), which lets you make any remote location look like a local one. It works well for making artifacts in an InterSystems IRIS namespace look like local files.
+The string `isfs` which appears in the **uri** for folders configured for server-side editing is an abbreviation created by InterSystems which stands for **InterSystems File Service**. It implements the VS Code [FileSystemProvider API](https://code.visualstudio.com/api/references/vscode-api#FileSystemProvider), which let's you make any remote location look like a local one. It works well for making artefacts in an InterSystems IRIS database namespace look like local files.
```

Comment:
```suggestion
The string `isfs` which appears in the **uri** for folders configured for server-side editing is an abbreviation created by InterSystems which stands for **InterSystems File Service**. It implements the VS Code [FileSystemProvider API](https://code.visualstudio.com/api/references/vscode-api#FileSystemProvider), which lets you make any remote location look like a local one. It works well for making artefacts in an InterSystems IRIS namespace look like local files.
```

Removed incorrect apostrophe. Also, I felt "database namespace" is confusing, particularly for developers as the two words have different precise meanings ("database" is where the code actually gets stored, "namespace" is a lens through which code interactions - including editing - usually occur).